app/ui/browser-blueprint/views/overview/overview-cards.jsx

```diff
@@ -32,7 +33,12 @@ const OverviewCards = function(props) {
       {props.pages.map((page, pageIndex) => (
         <OverviewCard key={`page-${page.id}`}
           page={page}
-          pageIndex={pageIndex}
+          width={UIConstants.CARD_WIDTH}
```

Comment:
It's not that they *should* have the same value, they just coincidentally do at the moment. I can easily see us needing different values for different frontends, but I agree that this isn't a requirement at the moment.